### DISCUSSION TITLE
Browser Mod Pop Up with Sonos Card

### DISCUSSION THREAD

**Discussion by psjnr84** [punxaphil/custom-sonos-card#543] [Q&A]
Hi All,

Do you happen to know if there is any support to have the Custom Sonos Card appear in the Browser Mod Pop up screen?

The reason for the ask is specifically so I can put a button on a particular room dashboard that pops up the Sonos card to make changes to the player but then closes once you’ve done that to maintain a tidy dashboard.

Any clue about whether that would work or perhaps using something like the Lovelace bubble card to do something similar?

**Comment by punxaphil** [comment]
Yes I've heard others succeeding with that.

**Reply by psjnr84** [reply]
I’ve managed to sort it finally. Works a treat. Parked the idea for the moment as I’ve created a better more compact dashboard and can fit things in better. Cheers for responding!
